javascript/computer_science/time_complexity.md

```diff
@@ -325,7 +325,7 @@ The following questions are an opportunity to reflect on key topics in this less
 - [What is Big O?](#what-is-big-o)
 - [What are the Big O Notations?](#big-o-notation)
 - [Why use Big O?](#why-big-o)
-- [What is Big Omega and why isn't it as useful?](#big--omega-notation)
+- [What is Big Omega and why isn't it as useful?](https://www.geeksforgeeks.org/analysis-of-algorithms-big-omega-notation/#when-to-use-bigomega-notation)
```

Comment:
Thanks for the PR. We just need to remove the extra dash from the fragment to get it to link to the correct subsection. The fragments are `#` followed by the ID of an element, and the link will scroll you to the matching element (or the top of the page if no match).

Our headings generate their IDs via a certain algorithm (lowercasing, removing punctuation then replacing spaces with single dashes) that also removes special characters. So the ID for the big omega heading is just `big-omega-notation` with no special characters. The only issue was the rogue extra dash

```suggestion
- [What is Big Omega and why isn't it as useful?](#big-omega-notation)
```

Reply:
Already commited the suggestion, thanks for the help